pkgs/applications/misc/gnuradio/default.nix

```diff
@@ -41,6 +41,7 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  patches = [ ./gnuradio-gcc7.patch ];
```

Comment:
Is there an external source for the patch? If possible, fetch it instead of checking it into the repository.

Reply:
The patch came from the Arch Linux packages git repo.

I have changed this PR to not use a patch, but instead apply its essential parts as `NIX_CFLAGS_COMPILE` and `substituteInPlace`.